glslc/src/file_compiler.cc

```diff
@@ -62,13 +62,25 @@ bool FileCompiler::CompileShaderFile(const std::string& input_file,
     source_string = string_piece(&(*input_data.begin()),
                                  &(*input_data.begin()) + input_data.size());
   }
-  StageDeducer deducer(input_file);
-  bool compilation_success =
-      Compile(source_string, shader_stage, error_file_name.str(), deducer,
-              glslc::FileIncluder(&include_file_finder_), output_stream,
-              &std::cerr, &total_warnings_, &total_errors_);
 
-  if (!compilation_success && output_stream->fail()) {
+  std::unique_ptr<FileIncluder> includer(new FileIncluder(&include_file_finder_));
+  options_.SetIncluder(std::move(includer));
+
+  shaderc::SpvModule result =
+      compiler_.CompileGlslToSpv(source_string.begin(), source_string.size(),
+                                 shader_stage, error_file_name.data(), options_);
+  total_errors_ += result.GetNumErrors();
+  total_warnings_ += result.GetNumWarnings();
+
+  bool compilation_success =
+      result.GetCompilationStatus() == shaderc_compilation_status_success;
+
+  // Write output to output file.
+  output_stream->write(result.GetData(), result.GetLength());
+  // Write error message to std::cerr.
+  std::cerr << result.GetErrorMessage();
+  // Something wrong happen for output.
```

Comment:
"happened on" instead of "happen for"

Also move the comment to the next line.